package.json

```diff
@@ -20,7 +20,19 @@
         "async"
     ],
     "version": "1.3.2",
```

Comment:
Wasn't sure how we wanted to bump the version for this change.

Reply:
the version should automagically get bumped via the standard changelog stuff 👍 , assuming your commits followed the appropriate conventions 😁

Reply:
Awesome :+1: